pkgs/by-name/cu/cursewords/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, python3Packages
+, fetchPypi
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "cursewords";
+  version = "1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Ng4UteSUG+hFF5iV/e4HNRpSe0voYSeJiHnGnPHXXA4=";
+  };
+
+  nativeBuildInputs = [
+    python3Packages.setuptools
+    python3Packages.wheel
+  ];
+
+  doCheck = false;
```

Comment:
can you add a comment, no tests, or failing tests ect

Reply:
Comment added. No tests, and when I added pytestCheckHook, it treats 0 tests as an error, so I disabled them instead.